lib/powerline.sh

```diff
@@ -105,7 +107,15 @@ __process_scripts() {
 		fi
 
 		if [ -n "$output" ]; then
-			powerline_segment_contents[$segment_index]=" $output "
+			if [ ${powerline_segment[4]} == "L" ] ; then
```

Comment:
`L` means 'output to the left of the space,
`R` means 'output to the right of the space,
`B` ?

I'm trying to understand the choice of letters :). It might not be so intuitive looking at a config and knowing what these does. 

And https://github.com/erikw/tmux-powerline/blob/0995e9a37214fae6e94d5c361b2ba73f84efe438/themes/default.sh#L51 would need an update

Reply:
OK all done!  I've improved the naming of the options so they're more easy to understand there.

The themes/default.sh file has been updated to include a pretty thorough usage example and information about the existing and new optional fields.